.github/workflows/test.yml

```diff
@@ -0,0 +1,47 @@
+name: Test
+on: [pull_request]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Read .nvmrc
+      run: echo ::set-output name=NVMRC::$(cat .nvmrc)
+      id: nvm
+
+    - name: Install Ubuntu packages
+      run: sudo apt-get install libxml2-dev libxslt1-dev python-dev
+
+    - name: Use Node.js
+      uses: actions/setup-node@v1
+      with:
+        node-version: '${{ steps.nvm.outputs.NVMRC }}'
+
+    - uses: actions/cache@v2
+      with:
+        path: ~/.npm
+        key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+        restore-keys: |
+          ${{ runner.os }}-node-
```

Comment:
Why does restore keys have a hanging `-`? Excuse my ignorance

Reply:
🤷 this is some magic I derived from https://github.com/actions/setup-node/issues/32#issuecomment-539794249 I suspect is just a clearer way of denoting that prefix.